contrib/labstack/echo.v4/option.go

```diff
@@ -109,3 +110,10 @@ func WithHeaderTags(headers []string) Option {
 		cfg.headerTags = internal.NewLockMap(headerTagsMap)
 	}
 }
+
+// WithErrorCheck sets the func which determines if err would be ignored (if it returns true, the error is not tagged)
```

Comment:
```suggestion
// WithErrorCheck specifies a function fn which determines whether the passed
// error should be marked as an error. The fn is called whenever a request
// finishes with an error.
```

(to use a similar description as we are providing for this option in other contribs)

Reply:
but I think it's important to tell users what happens if this func returns true/false

Reply:
hey! @mrkagelui sorry I missed this conversation.
The comments don't necessary need to align (I think clarifying what true/false means is a good idea), it's just the logic of what the boolean means flipped with the name change (`WithErrorCheck` will mark the error as error when returns `true`, and ignore it when it returns `false`).

Reply:
ok, so you are saying that since we call it `WithErrorCheck`, we'd mark it error when it's true, will make the change